### PR TITLE
Feature/code cleanups

### PR DESCRIPTION
**Pull request by pi3k14**
I did some cleanup of the code you might be interested in, happy to strip down to a subset if you think this is "over the top" :)

---
Rebus is [MIT-licensed](https://opensource.org/licenses/MIT). The code submitted in this pull request needs to carry the MIT license too. By leaving this text in, __I hereby acknowledge that the code submitted in the pull request has the MIT license and can be merged with the Rebus codebase__.

### CODE REVIEW

File: Rebus.SqlServer.Tests/Assumptions/TestSpWho.cs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Linq;
+
```
Comment (mookid8000) [inline Rebus.SqlServer.Tests/Assumptions/TestSpWho.cs:3]: why?

File: Rebus.SqlServer.Tests/Assumptions/TestTableName.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Text.RegularExpressions;
+
```
Comment (mookid8000) [inline Rebus.SqlServer.Tests/Assumptions/TestTableName.cs:3]: why?

File: Rebus.SqlServer.Tests/Bugs/TestBugWhenCustomizingExpiredMessagesCleanupInterval.cs [hunk 3]
```diff
@@ -1,6 +1,8 @@
 ﻿using System;
 using System.Threading.Tasks;
+
```
Comment (mookid8000) [inline Rebus.SqlServer.Tests/Bugs/TestBugWhenCustomizingExpiredMessagesCleanupInterval.cs:3]: why?

File: Rebus.SqlServer.Tests/Bugs/TestBugWhenCustomizingExpiredMessagesCleanupInterval.cs [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
 using NUnit.Framework;
+
```
Comment (mookid8000) [inline Rebus.SqlServer.Tests/Bugs/TestBugWhenCustomizingExpiredMessagesCleanupInterval.cs:5]: why?

File: Rebus.SqlServer.Tests/Bugs/TestBugWhenFinishingSagaAndAuditingIsEnabled.cs [hunk 5]
```diff
@@ -2,7 +2,9 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+
```
Comment (mookid8000) [inline Rebus.SqlServer.Tests/Bugs/TestBugWhenFinishingSagaAndAuditingIsEnabled.cs:5]: why?

File: Rebus.SqlServer.Tests/Bugs/TestBugWhenFinishingSagaAndAuditingIsEnabled.cs [hunk 6]
```diff
@@ -2,7 +2,9 @@
...
 using NUnit.Framework;
+
```
Comment (mookid8000) [inline Rebus.SqlServer.Tests/Bugs/TestBugWhenFinishingSagaAndAuditingIsEnabled.cs:7]: etc.... 🙄

Comment (pi3k14) [reply Rebus.SqlServer.Tests/Bugs/TestBugWhenFinishingSagaAndAuditingIsEnabled.cs:7]: Well, it has become current standard to place system using's at the top and use blank lines between each group (alphabetically listed after system).

**Comment by pi3k14** [comment]
It's your call, but you might wan't to have a look at the other commits in the PR.

(no offence taken :) )

**Comment by mookid8000** [comment]
If the blind code cleanup is in a separate commit, you're most welcome to revert that commit and send a new PR 😺

**Pull request closed without merging**